### PR TITLE
{bio}[foss/2016b] preseq 2.0.2

### PR DESCRIPTION
**Pull request by SethosII**
This adds preseq.

### CODE REVIEW

File: easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'preseq'
+version = '2.0.2'
+
+homepage = 'http://smithlabresearch.org/software/%(name)s'
+description = """Software for predicting library complexity and genome coverage in
+ high-throughput sequencing."""
```
Comment (boegel) [inline easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb]: style nitpicking: this fits on a single line with < 120 chars

```python
description = "Software for predicting library complexity and genome coverage in high-throughput sequencing."

File: easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/smithlabcode/%(name)s/releases/download/v%(version)s']
+sources = ['%(name)s_v%(version)s.tar.bz2']
+
+checksums = ['1d7ea249bf4e5826e09697256643e6a2473bc302cd455f31d4eb34c23c10b97c']
+
```
Comment (boegel) [inline easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb]: more style nitpicking: drop empty line above

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/861dd407c11eb428ec67efe1f48392e0 for a full test report.

**Comment by boegel** [comment]
@SethosII Missing dependency on `GSL`? See failing test report...

File: easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb]: @boegel I changed this.

File: easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb (hunk 2 above)
Comment (SethosII) [reply easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb]: @boegel I changed this.

**Comment by SethosII** [comment]
@boegel I added `GSL` as a dependency.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2470.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ae72268e0418c89807fbea2937ca2150 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2125.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1626c295414fbd0aaa52fbe2940e1fce for a full test report.

File: easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+description = "Software for predicting library complexity and genome coverage in high-throughput sequencing."
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/smithlabcode/%(name)s/releases/download/v%(version)s']
+sources = ['%(name)s_v%(version)s.tar.bz2']
+checksums = ['1d7ea249bf4e5826e09697256643e6a2473bc302cd455f31d4eb34c23c10b97c']
+
+files_to_copy = [(['bam2mr', '%(name)s'], 'bin'), 'gpl-3.0.txt']
```
Comment (boegel) [inline easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb]: @SethosII: final style nitpicking remark: please move this down below `dependencies`

Comment (SethosII) [reply easybuild/easyconfigs/p/preseq/preseq-2.0.2-foss-2016b.eb]: @boegel I changed this.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2488.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/639bd51f827b0858c93f5f56696200c6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b2b97e12ea038c85b1e545297363c29b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**